plugins/CoreAdminHome/templates/pluginSettings.twig

```diff
@@ -13,12 +13,12 @@
         <h2 piwik-enriched-headline>{{ 'CoreAdminHome_SystemPluginSettings'|translate }}</h2>
     {% endif %}
 
-    <p>
+    <div>
         {{ 'CoreAdminHome_PluginSettingsIntro'|translate }}
         {% for pluginName, settings in pluginsSettings %}
             <a href="#{{ pluginName|e('html_attr') }}">{{ pluginName }}</a>{% if not loop.last %}, {% endif %}
         {% endfor %}
-    </p>
+    </div>
```

Comment:
why change a paragraph to a div? Visually and semantically it's a paragraph:

![capture d ecran 2015-06-18 a 11 21 36](https://cloud.githubusercontent.com/assets/720328/8228062/3a89fa3e-15ac-11e5-9091-67775108329f.png)